### PR TITLE
Correct email

### PR DESCRIPTION
**Pull request by loneil**
# Description

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/frontend/src/components/agriseafoodopscreening/Landing.vue [hunk 1]
```diff
@@ -144,8 +144,8 @@
                   <br />
                   <br />Toll-free: 1-877-533-1789 | Email:
                   <a
-                    href="mailto:AGRI.AgricultureandSeafoodOperator.IPCP@gov.bc.ca"
-                  >AGRI.AgricultureandSeafoodOperator.IPCP@gov.bc.ca</a>
+                    href="Contact@AgSafeBC.ca"
```
Comment (jujaga) [inline app/frontend/src/components/agriseafoodopscreening/Landing.vue:147]: Is this supposed to be a mailto: link?

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-vmvfjv-tools.pathfinder.gov.bc.ca/dashboard?id=common-forms-toolkit-pr-108

File: app/frontend/src/components/agriseafoodopscreening/Landing.vue (hunk 1 above)
Comment (loneil) [reply app/frontend/src/components/agriseafoodopscreening/Landing.vue]: Accidentally removed it, it's back now

**Comment by jujaga** [comment]
> 
> 
> i'm guessing we dont worry about bots scraping email adresses. ie no need to obfuscate or display using js and such.

It might be something worth investigating down the line. Let's create a backlog story ticket to consider this in the future. As of right now I suspect it should be fine as is.

**Pull request merged**